### PR TITLE
Feature/server offline process

### PR DESCRIPTION
**Pull request by hirsch88**

### CODE REVIEW

File: src/components/ConnectionHandler.vue [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  }
+
+}
+</script>
+
+<style scoped lang="scss">
+@import "../styles/colors";
+
+div.server-unavailable, div.offline {
+  .fa-times{
```
Comment (keniseli) [inline src/components/ConnectionHandler.vue]: Is this formatted properly?

File: src/locals/de.ts [hunk 2]
```diff
@@ -53,4 +54,14 @@ export default {
     login_failed: 'Username oder Passwort ist falsch.',
     login_incomplete: 'Bitte geben Sie Username und Passwort an.',
   },
+  connection: {
+    server_unavailable: {
```
Comment (keniseli) [inline src/locals/de.ts:58]: Is it common practice to use the underscore here?

**Review by keniseli — APPROVED** [review]

File: src/locals/de.ts (hunk 2 above)
Comment (hirsch88) [reply src/locals/de.ts:58]: is

File: src/components/ConnectionHandler.vue (hunk 1 above)
Comment (hirsch88) [reply src/components/ConnectionHandler.vue]: no

**Pull request merged**